### PR TITLE
Feature: Remove Items

### LINKED ISSUE #49: 4b. Enable users to remove ingredients from Fridget

### PR DESCRIPTION
**Pull request by alvynben**
Fixes #49 .

### CODE REVIEW

File: src/main/java/seedu/ui/Ui.java [hunk 1]
```diff
@@ -65,6 +65,26 @@ public void printReactionToAddingIngredient(Ingredient ingredient) {
...
+        printLine("Which item would you like to be removed?");
+        printListOfIngredients(matchingItems,true);
+        printSeparatorLine();
+
+        // TODO: Needs to catch incorrect index and inputs
+        int index = Integer.parseInt(readUserInput());
+        printSeparatorLine();
+
+        return index;
+
```
Comment (zonglun99) [inline src/main/java/seedu/ui/Ui.java:85]: can remove this extra space

File: src/main/java/seedu/commands/RemoveCommand.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                ui.printReactionToRemovingIngredient(itemToBeRemoved);
+            }
+        }
+
+        // TODO: Catch if index is invalid
+        int indexOfItemToBeRemoved = ui.getIndexOfItemToBeRemoved(matchingItems) - 1;
+        Ingredient itemToBeRemoved = matchingItems.get(indexOfItemToBeRemoved);
+        ingredientList.removeIngredient(itemToBeRemoved);
+        ui.printReactionToRemovingIngredient(itemToBeRemoved);
+
```
Comment (zonglun99) [inline src/main/java/seedu/commands/RemoveCommand.java:33]: this space can remove also

File: src/main/java/seedu/commands/RemoveCommand.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package seedu.commands;
+
+import seedu.data.ingredient.Ingredient;
+import seedu.parser.Parser;
+import seedu.storage.IngredientList;
+import seedu.ui.Ui;
+
+import java.util.ArrayList;
+
+public class RemoveCommand extends Command{
```
Comment (zonglun99) [inline src/main/java/seedu/commands/RemoveCommand.java:10]: Got error here, space between Command and  {

**Review by zonglun99 — APPROVED** [review]

File: src/main/java/seedu/ui/Ui.java (hunk 1 above)
Comment (alvynben) [reply src/main/java/seedu/ui/Ui.java:102]: Have changed the function, so not applicable.

File: src/main/java/seedu/commands/RemoveCommand.java (hunk 3 above)
Comment (alvynben) [reply src/main/java/seedu/commands/RemoveCommand.java]: Resolved.

**Pull request merged**